lbcpresenter/src/commonMain/kotlin/studio/lunabee/compose/foundation/presenter/LBPresenter.kt

```diff
@@ -25,15 +25,9 @@ import androidx.compose.runtime.Composable
 import androidx.compose.runtime.LaunchedEffect
 import androidx.compose.runtime.getValue
 import androidx.lifecycle.ViewModel
-import androidx.lifecycle.compose.collectAsStateWithLifecycle
 import androidx.lifecycle.viewModelScope
-import kotlinx.coroutines.flow.Flow
-import kotlinx.coroutines.flow.MutableSharedFlow
-import kotlinx.coroutines.flow.MutableStateFlow
-import kotlinx.coroutines.flow.SharingStarted
-import kotlinx.coroutines.flow.StateFlow
-import kotlinx.coroutines.flow.onEach
-import kotlinx.coroutines.flow.stateIn
+import kotlinx.coroutines.flow.*
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/LunabeeStudio/Lunabee_Compose_Android/blob/5989407f62483b42f4b645531bb140b231c552e0/lbcpresenter/src/commonMain/kotlin/studio/lunabee/compose/foundation/presenter/LBPresenter.kt#L29"></span>kotlinx.coroutines.flow.* is a wildcard import. Replace it with fully qualified imports.</td>
    </tr>
  </tbody>
</table>